lib/matplotlib/pyplot.py

```diff
@@ -968,6 +968,10 @@ def draw():
 
     This is equivalent to calling ``fig.canvas.draw_idle()``, where ``fig`` is
     the current figure.
+
+    See Also
+    --------
+    .draw_idle
```

Comment:
```suggestion
    .FigureCanvasBase.draw_idle
    .FigureCanvasBase.draw
```

Doing two things:
1. include the class to help clearify that these are methods not another function
2. add `draw` to help highlight that there are other `draw` methods.

Reply:
Merged in origin commit